index.js

```diff
@@ -63,6 +73,35 @@ module.exports = {
         'ember-cli-terser',
         // Linting
         '@babel/plugin-proposal-decorators',
+
+        // Upstream TypeScript blueprint is out of date, but
+        // There is concensus on removing all this from the upstream blueprint as well.
+        '@types/ember',
+        '@types/ember-data',
+        '@types/ember-data__adapter',
+        '@types/ember-data__model',
+        '@types/ember-data__serializer',
+        '@types/ember-data__store',
+        '@types/ember__application',
+        '@types/ember__array',
+        '@types/ember__component',
+        '@types/ember__controller',
+        '@types/ember__debug',
+        '@types/ember__destroyable',
+        '@types/ember__engine',
+        '@types/ember__error',
+        '@types/ember__helper',
+        '@types/ember__modifier',
+        '@types/ember__object',
+        '@types/ember__owner',
+        '@types/ember__polyfills',
+        '@types/ember__routing',
+        '@types/ember__runloop',
+        '@types/ember__service',
+        '@types/ember__string',
+        '@types/ember__template',
+        '@types/ember__test',
+        '@types/ember__utils',
```

Comment:
I feel like we should only be removing these if you've passed --typescript 🤔

Reply:
done